src/components/MainPage/MainPage.module.css

```diff
@@ -0,0 +1,122 @@
+@media (min-width: 1084px) {
+    .intro {
+        position: relative;
+    }
+
+    .introContent {
+        margin-right: 632px;
+    }
+
+    .intro::after {
+        display: block;
+        content: '';
+        position: absolute;
+        top: 0;
+        right: 0;
+        width: 632px;
+        height: 798px;
+        background: url("./logo.png") no-repeat top center;
```

Comment:
2.5mb image seems too big, maybe we can change it to GatsbyImage, or at least compress ?